### PR TITLE
docs: added slack community guidelines

### PR DESCRIPTION
**Pull request by srinify**
Small change - just adding Slack community guidelines in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,6 +109,40 @@ supported databases can be found
...
+- We strive to make this community a warm place for people from all industries, use cases, geographies, and backgrounds. Harassment of any kind is not acceptable and won’t be tolerated. 
+- Please follow the guidelines as outlined in the Superset Community code of conduct here.
+
+**2. Use the right channel.**
+
+- Channels are an effective way to organize and focus discussions while also empowering members to opt-in to the types of content they’re interested in. When questions are posted or discussions are started in the wrong channel, it dilutes the trust of the members in the channel and, more practically, makes it harder for your questions to be answered.  
+
+**3. Ask thoughtful questions.**
+
+- We’re all here to help each other out. The best way to get help is by investing effort into your questions. First check and see if your question is answered in [the Superset documentation](https://superset.incubator.apache.org/faq.html) or on [Stack Overflow](https://stackoverflow.com/search?q=apache+superset). Then, use Slack search to see if your question has already been asked and answered in the past. If you still feel the need to ask a question, make sure you include:
```
Comment (willbarrett) [inline README.md:127]: Maybe also link to the Github Issues, both open and closed?

**Review by willbarrett — APPROVED** [review]
Looks good to me, one additional suggestion for a place to search for answers.

File: README.md [hunk 2]
```diff
@@ -109,6 +109,40 @@ supported databases can be found
...
 * [Mailing list](https://lists.apache.org/list.html?dev@superset.apache.org)
 
+## Slack Community Guidelines
+
+If you decide to join the [Community Slack](https://join.slack.com/t/apache-superset/shared_invite/enQtNDMxMDY5NjM4MDU0LWJmOTcxYjlhZTRhYmEyYTMzOWYxOWEwMjcwZDZiNWRiNDY2NDUwNzcwMDFhNzE1ZmMxZTZlZWY0ZTQ2MzMyNTU), please adhere to the following rules:
+
+**1. Treat everyone in the community with respect.**
+
+- We strive to make this community a warm place for people from all industries, use cases, geographies, and backgrounds. Harassment of any kind is not acceptable and won’t be tolerated. 
+- Please follow the guidelines as outlined in the Superset Community code of conduct here.
```
Comment (etr2460) [inline README.md:119]: can you add a link to the code of conduct? Something like:

```
Please follow the guidelines as outlined in the [Superset Community code of conduct](https://github.com/apache/incubator-superset/blob/master/CODE_OF_CONDUCT.md)
```

File: README.md [hunk 3]
```diff
@@ -109,6 +109,40 @@ supported databases can be found
...
+
+**5. Communicate openly**
+
+- Unless you have explicit permission from the person, please avoid sending direct messages to individuals. Communicating in public channels ensures that we’re all respecting each other’s attentions and we can scalably moderate our communication to mitigate harassment or discrimination. Do not use direct messages to pitch products and services. If you are receiving unwelcome direct messages, please notify an admin.
+
+**6. Practice good Slack hygiene by using threads for discussions and emojis for light reactions.**
+
+- The medium is the message. Slack can foster a warm, collaborative, and organized community when used effectively. We want to respect people’s attentions (thread notifications > channel notifications > DM notifications) and we want to  improve information density (a member should be able to browse and explore many convo threads, not just see one thread discussed in a top level channel).
```
Comment (etr2460) [inline README.md:143]: nit, you've got a double space between `to` and `improve`

**Review by etr2460 — APPROVED** [review]
a couple nits, otherwise lgtm

File: README.md (hunk 1 above)
Comment (srinify) [reply README.md:127]: Generally I agree with this, but boy is Github issues a rat's nest right now! Let me find the right way to phrase it here

**Comment by mistercrunch** [comment]
Seems like a lot of text for the README, maybe that should be a section of the code of conduct?

**Pull request merged**